kitsune/wiki/utils.py

```diff
@@ -194,3 +212,51 @@ def get_visible_document_or_404(
 
 def get_visible_revision_or_404(user, **kwargs):
     return get_object_or_404(Revision.objects.visible(user, **kwargs))
+
+
+def build_topics_data(request, product, topics):
```

Comment:
Let's add type hints similar to what you did with the product view.

Reply:
Done.